src/java.desktop/share/classes/sun/print/resources/serviceui_de.properties

```diff
@@ -63,7 +63,7 @@ label.pstype=Typ:
 label.rangeto=Bis
 label.size=G&röße:
 label.source=&Quelle:
-label.outputbins=A&usgabefächer:
```

Comment:
How confident are we in this ?
google translate says that the old text referred to 'compartments' and the new one to 'areas' and I wonder why that is better ?

https://translate.google.com/?sl=auto&tl=en&text=Ausgabebereiche%0AAusgebefacher&op=translate

Reply:
"Ausgabefächer" is the best German translation here.

Reply:
The reason for going with "Bereich (area)" instead of "Fach (compartment)" is that an area is more general compared to a compartment. At least in German: every "Fach" is a "Bereich", but every "Bereich" is NOT a "Fach".

Reply:
Thanks for the input @sormuras!